integration/testdata/build-dependencies/skaffold.yaml

```
@@ -13,7 +13,7 @@ build:
         SLEEP: "0"
         FAIL: "0"
     requires:
-    - image: image2
+    - image: image2 #test build-arg set to image field, when alias field is missing (`ARG image2`)
```

I was a bit confused when first reading the comment
```suggestion
    - image: image2 #test that build-arg set to image field when alias field is missing (`ARG image2`)
```